api/python/t4/packages.py

```diff
@@ -262,6 +262,8 @@ def capture(self, path, prefix=None):
             logical_key = prefix + f.relative_to(src_path).as_posix()
             # TODO: Warn if overwritting a logical key?
             pkg = pkg.set(logical_key, entry)
+
+        pkg._unset_tophash()
```

Comment:
Can you add an inline comment about why you have to do this and also explain it in the PR description?